src/ui/ListView.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+var style = require('./styles/basicStyles')
+var yo = require('yo-yo')
+var ui = require('../helpers/ui')
+var DropdownPanel = require('./DropdownPanel')
+
+function ListView (_parent, _codeManager) {
```

Comment:
ListView sounds like a very generic UI element name, but it seems to be very specific here